server.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+const express = require('express');
+
+const PORT = 4000;
+
+const app = express();
+
+
+app.use(express.static('./public'));
+
+app.use(function(request, response, next){
```

Comment:
Good attempt at using error middleware here. Of course, since everything works and you have no errors, we wouldn't see it. Refer back to lecture code videos of when Sam showed us what he did to get his 404 page to appear.